### DISCUSSION TITLE
Validate Token

### DISCUSSION THREAD

**Discussion by psrinivasan9128** [rajanadar/VaultSharp#250] [Q&A]
Hi 
I am trying to use cubby hole way of reusing the token. I am able to save and extract token from cubby hole. However if a token is expired, i am not sure how to validate it.

I am using C# as my code base, is there any API method, i can look and use or should i save the TTL file and compare it against the current time to validate the token

thanks

**Comment by rajanadar** [comment]
hi @psrinivasan9128 , do you still see this issue? If yes, can you please create a GH issue with the details
